README.md

```diff
@@ -1,82 +1,67 @@
 # Deferred
 
-This is an implementation of [OCaml's Deferred](https://ocaml.janestreet.com/ocaml-core/111.25.00/doc/async_kernel/#Deferred) for Swift.
-
-## Vital Statistics
-- What: Lets you work with values that haven't been determined yet,
-        like an array that's coming (one day!) from a web service call.
-- Who: John Gallagher <jgallagher@bignerdranch.com> wrote this.
-- Swift: 2.2
-- License: MIT
-- Carthage: Yup.
-- CocoaPods: As [`BNRDeferred`](https://cocoapods.org/pods/BNRDeferred).
-
-## Installation
-
-### [Carthage]
-
-[Carthage]: https://github.com/Carthage/Carthage
-
-Add the following to your Cartfile:
-
-```ruby
-github "bignerdranch/Deferred" "2.0b6"
-```
-
-Then run `carthage update`.
-
-Follow the current instructions in [Carthage's README][carthage-installation]
-for up to date installation instructions.
-
-[carthage-installation]: https://github.com/Carthage/Carthage/blob/master/README.md
-
-### [CocoaPods]
-
-[CocoaPods]: http://cocoapods.org
-
-Add the following to your [Podfile](http://guides.cocoapods.org/using/the-podfile.html):
-
-```ruby
-pod 'BNRDeferred'
-```
-
-You will also need to make sure you're opting into using frameworks:
-
-```ruby
-use_frameworks!
-```
-
-Then run `pod install`.
+This is an implementation of [OCaml's Deferred](https://ocaml.janestreet.com/ocaml-core/111.25.00/doc/async_kernel/#Deferred) for Swift. It lets you work with values that haven't been determined yet, like an array that's coming later (one day!) from a web service call.
+
+### Vital Statistics
+
+|             | **Status**                                                                                                                                                  |
+|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
I think using the footnote-style version would work, like:

[mumble][1]

  [1]: url goes here

That can be a string rather than a 1, and I recommend that in order to
avoid conflicts. :) You can stick the link-footnotes right under the table
so they stay by the text using them.

--
Jeremy W. Sherman <jeremy@bignerdranch.com>

2016-04-12 13:50 GMT-04:00 Zachary Waldowski <notifications@github.com>:

> In README.md
> <https://github.com/bignerdranch/Deferred/pull/94#discussion_r59421458>:
>
> > -pod 'BNRDeferred'
> > -```
> > -
> > -You will also need to make sure you're opting into using frameworks:
> > -
> > -```ruby
> > -use_frameworks!
> > -```
> > -
> > -Then run `pod install`.
> > +This is an implementation of [OCaml's Deferred](https://ocaml.janestreet.com/ocaml-core/111.25.00/doc/async_kernel/#Deferred) for Swift. It lets you work with values that haven't been determined yet, like an array that's coming later (one day!) from a web service call.
> > +
> > +### Vital Statistics
> > +
> > +|             | **Status**                                                                                                                                                  |
> > +|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
>
> Make the width *really* big. [image: :grimacing:]
>
> (Maybe I can use a URL shortener.)
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/bignerdranch/Deferred/pull/94/files/b56bbd18e89e5b391461bb520073b5f226dfdb9a#r59421458>
>